actions/subscribers.go

```diff
@@ -436,16 +424,16 @@ func ImportSubscribers(c *gin.Context) {
 		return
 	}
 
-	go func(ctx context.Context, userID int64, segs []entities.Segment, r io.ReadCloser) {
-		imp := importer.NewS3SubscribersImporter(client)
-		err := imp.ImportSubscribersFromFile(ctx, u.ID, segs, r)
+	go func(ctx context.Context, filename string, userID int64, segs []entities.Segment, r io.ReadCloser) {
+		svc := subscribers.New(s3Client, storage.GetFromContext(c))
+		err := svc.ImportSubscribersFromFile(ctx, filename, u.ID, segs, r)
 		if err != nil {
 			logger.From(ctx).WithFields(logrus.Fields{
-				"filename": reqParams.Filename,
+				"filename": filename,
 				"segments": segs,
 			}).WithError(err).Warn("Unable to import subscribers.")
 		}
-	}(c, u.ID, segs, res.Body)
+	}(c, reqParams.Filename, u.ID, segs, res.Body)
```

Comment:
@FilipNikolovski @Dzalevski are we safe to use s3client in the go routine without sending copy of it

Reply:
We can send it as an argument in the function along with the rest of them.